books/u-physics/bake

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+require 'bundler/inline'
+
+gemfile do
+  gem 'openstax_kitchen', ENV['USE_LOCAL_KITCHEN'] ? { path: '/code/kitchen' } : '2.0.0'
+  gem 'slop', '4.8.2'
+  gem 'byebug'
+end
+
+before = Kitchen::BookRecipe.new(book_short_name: :u_physics) do |doc|
+  include Kitchen::Directions
+
+  book = doc.book
+
+  # remove exercises (take out once BakeExercise works)
+  book.search('[data-type="exercise"]').trash
+end
+
+recipe = Kitchen::BookRecipe.new(book_short_name: :uphysics) do |doc|
+  include Kitchen::Directions
+
+  book = doc.book
+
+  book.search('cnx-pi').trash
+
+  BakePreface.v1(book: book)
+  BakeUnitTitle.v1(book: book)
+
+  book.chapters.each do |chapter|
+    BakeChapterGlossary.v1(chapter: chapter, metadata_source: book.metadata)
+    BakeChapterKeyEquations.v1(chapter: chapter, metadata_source: book.metadata)
+    BakeChapterKeyConcepts.v1(chapter: chapter, metadata_source: book.metadata)
+  end
+
+  BakeChapterIntroductions.v1(book: book)
+  BakeChapterTitle.v1(book: book)
+
+  book.chapters.each do |chapter|
+    BakePageAbstracts.v1(chapter: chapter)
+
+    chapter.tables('$:not(.unnumbered)').each do |table|
+      BakeNumberedTable.v1(table: table,
+                           number: "#{chapter.count_in(:book)}.#{table.count_in(:chapter)}")
+    end
+
+    chapter.examples.each do |example|
+      BakeExample.v1(example: example,
+                     number: "#{chapter.count_in(:book)}.#{example.count_in(:chapter)}",
+                     title_tag: 'h3')
+    end
+
+    chapter.pages('$:not(.introduction)').each do |page|
+      page.search("div[data-type='description']").each(&:trash)
+      page.add_class('chapter-content-module')
+
+      title = page.search("div[data-type='document-title']").first
+      title.name = 'h2'
+      title.replace_children(with:
+        <<~HTML
+          <span class="os-number">#{chapter.count_in(:book)}.#{page.count_in(:chapter)}</span>
+          <span class="os-divider"> </span>
+          <span data-type="" itemprop="" class="os-text">#{title.text}</span>
+        HTML
+      )
+    end
```

Comment:
`BakeNonIntroductionPage` is a thing now, can use here

honestly I should have asked @KarinaMendez2  to make it `BakeNonIntroductionPages` plural and have it contain the `chapter.pages` loop too, since it assumes `page.count_in(:chapter)` exists and really will only ever be called inside such a loop.

Reply:
I'll do a PR about this :)